src/sort-lines.ts

```diff
@@ -27,25 +37,16 @@ function sortLines(textEditor: vscode.TextEditor, startLine: number, endLine: nu
     removeBlanks(lines);
   }
 
-  lines.sort(algorithm);
-
-  if (removeDuplicateValues) {
-    removeDuplicates(lines, algorithm);
-  }
+  lines = transformers.reduce((lines, transform) => transform(lines), lines);
```

Comment:
It's a little confusing how transformers is a global and a function param.

Reply:
very fair point :+1: 
I could rename the global `transformers` to `transformerSequences` unless you have another name in mind?

Reply:
And something like [this tslint rule](https://palantir.github.io/tslint/rules/no-shadowed-variable/) might help to avoid this kind of mistakes, I'll add it if you don't mind.

Reply:
Yeah that change would be fine too, it was especially confusing how the global `transformers` was a different shape but very similar.

I was thinking the same thing with tslint, please add 😃

Reply:
just added the changes 👍